### DISCUSSION TITLE
Audio advice for a newbie

### DISCUSSION THREAD

**Discussion by Psi58** [GrandOrgue/grandorgue#1714] [Help]
I couldn't be happier with the performance of my headphones (Superlux HD-330), but the sound from speakers is muddy, slightly 'boomy' at lower frequencies and lacks brilliance. This is the situation:

- Room : approx 3m x 2m, wooden floor and shutters (ie no carpet or curtains)
- Audio interface : Behringer UMC204HD
- Speakers : M-Audio BX-5 D3, sitting directly on the organ console, ie about shoulder height

Options I am considering:
(1) Fixing the speakers nearer the ceiling. This would be non-trivial (cable routing etc...), so I'd like to be pretty confident it would make a significant improvement before doing this.
(2) Adding an equaliser. I presume that this would have to be a software equaliser on the PC (before the USB audio is input to the Behringer) because there appears to be no way to route that signal through an 'insert' to the UMC204HD, based on the diagram [here ](https://audiosciencereview.com/forum/index.php?threads/behringer-umc204-hd-audio-interface-review.9856/post-717115).
(3) Adding a sub-woofer. Hopefully this would offload the lower frequencies from the monitors, allowing them to do a better job with the mid/high frequencies. Due to limited budget (~£250) and space (see above) I'm not expecting to get the full 16Hz rumble, but I'd hope it would clean up the output somewhat.

So... what do you think? Which of these would make the most difference? And, if (2) or (3) are recommended, what models do you recommend? Many thanks!

**Comment by ClarkeBrunt** [comment]
Hi. I'm certainly not an audio expert, so can only relate my own experience.

We've got very similar equipment, mine being either what was supplied by the chap who midified the console for me, or his suggestions. I wonder if we got our consoles from the same place??

My audio interface is also Behringer but the UMC404HD - I think the initial 4 is about the number of inputs, which I'm not using anyway, so pretty much the same as yours. There's a diagram of the operation of the '4' variant of the Behringer here https://github.com/mcnster/umc-404hd-signal-path/ which looks much the same as your diagram.

I've got the same M-Audio speakers, but I added a sub-woofer.

My experience too was that it sounds very bass-heavy. Could this be because these are 'monitor' speakers (whatever exactly that means)? Also some individual low notes would 'boom' - more likely room resonance - I 'voiced' some of these to be quieter in GO.

Although it was supplied with a second-hand computer with Windows 10, I replaced it with Ubuntu Studio (Linux), after testing (with a bootable USB stick) that the hardware was recognised and worked. Maybe the touchscreen doesn't work as well as on Windows (I don't know how to get right-click), but it's fine for GO, and I can always use the mouse for anything fiddly.

Using just the software that was pre-installed on Ubuntu Studio, I set up a graphic-equaliser, using Carla as software patchbay, and one of the many plugins as the equaliser. I use Jack to connect the various audio things together, and start the various things automatically on bootup - so I'm pretty happy with it. Other audio applications also play through this same route, so the equaliser affects them too.

As I wrote over here #1407 (before changing to Linux) the computer is 'just' a Dell Optiplex 7010 (small form factor), an i3-3240 @ 3.40 GHz with 16GB RAM, so really nothing special at all, yet seems to work fine and loads the Friesach organ (just).

If you're on Windows, then I don't know what options there are for an equaliser, but I expect something is available.

**Reply by Psi58** [reply]
Very helpful, thanks. I suspect this is quite a small community, so the chances of us having bought equipment or received advice from the same sources are fairly high!

I've added an equalizer to my PC, with encouraging initial results. For some reason I don't fully understand it doesn't work with the Behringer ASIO driver, so I've reverted to PortAudio:Windows DirectSound, which works but adds noticeable latency. More experimentation needed...

My experience with different computers is similar - the processor and bus speed are not that critical, but the amount of RAM can be limiting. I'm using a laptop with 16GB RAM, which I'll probably replace with a desktop that can accommodate 64GB.
